crates/api/src/op_store.rs

```diff
@@ -70,6 +90,12 @@ pub trait OpStore: 'static + Send + Sync + std::fmt::Debug {
         end: Timestamp,
     ) -> BoxFuture<'_, K2Result<Vec<OpId>>>;
 
+    /// Read a list of ops from the store.
+    fn read_ops(
```

Comment:
The DHT diff PR defines `retrieve_ops`, we'll have to resolve these two

Reply:
resolved